gitlab/cmd/controller/main.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2020 The TriggerMesh Authors.
```

Comment:
Should the copyright header also be updated here?

Reply:
ok, thanks, didn't know that we should change it